sdk/eventhub/event-hubs/changelog.md

```diff
@@ -1,3 +1,11 @@
+### Coming Soon 2.1.0
+
+- Use of Websockets and support for proxy environment added. 
```

Comment:
Added support for WebSockets. WebSockets enable Event Hubs to work over an HTTP proxy and in environments where the standard AMQP port 5671 is blocked. Refer to the [useProxy](...) sample to see how to use WebSockets. If a proxy is not needed, do not set the proxy-related configurations to connect directly over a WebSocket.